onnxruntime/core/platform/windows/env.cc

```diff
@@ -113,6 +131,11 @@ class WindowsThread : public EnvThread {
 
 class WindowsEnv : public Env {
  public:
+
+  explicit WindowsEnv() {
```

Comment:
Can you elaborate on why `explicit` is needed here?

Reply:
Explicit is not needed.